### ISSUE TITLE
Huge growth of records from Proxy address in layer 7 monitoring in RC 1.4.0

### ISSUE THREAD

**Issue by fabianbur** [roundcube/roundcubemail#7033]
With the correction of issuess #7010 #6800  #5663 Roundcube functionality in Haproxy was corrected with monitoring in layer 7. However, I now notice unusual behaviour in the session table, which did not appear in Roundcube 1.3x: the session table is full of open sessions from the reverse proxy address. More than 1000 records in fifteen minutes, only from the proxy address. Those are 4000 records per hour and 96000 per day. I don't know what the maximum of the table is, but I think it could have unexpected consequences.

Apache2 Roundcube records are also growing tremendously.

Maybe an option that prevents any registration from the proxy's ip address?

**Comment by alecpl** [comment]
I don't really know what that could be. The session on login page access was also created in 1.3. Also, old sessions will be removed at some point, so it should not grow infinitely.

What PHP version? What session engine in Roundcube?

**Comment by fabianbur** [comment]
I spent the whole afternoon trying out an old virtual machine with roundcube 1.1, which I updated to 1.3 and then to 1.4  without any of the problems detected.

I realized that I had another site in apache2 by default and that must have been the solution from the beginning. Layer 7 monitoring goes to that null site, and works even with roundcube 1.4.0 without any patch.

When leaving only the roundcube site activated, it is the default site and fill grow in the session table also with roundcube 1.3

Apologies for opening the incident, but it helped me learn something new. I proceed to close it.

Regards
